lib/takeaway.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'menu'
+require_relative 'order'
+
+class Takeaway
+
+  attr_reader :menu
+
+  def initialize
+    @menu = Menu.new
+    @order = Order.new
+  end
+
+  def show_menu
+    menu
+  end
+
+  def place_order(item, qty)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `item`. If it's necessary, use `_` or `_item` as an argument name to indicate that it won't be used. You can also write as `place_order(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)<br>Lint/UnusedMethodArgument: Unused method argument - `qty`. If it's necessary, use `_` or `_qty` as an argument name to indicate that it won't be used. You can also write as `place_order(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)